source/best-practices/file-organization.md

```diff
@@ -33,7 +33,7 @@
 
 - 为了满足日常科研需求，在家目录下新建的、用于存储科研相关文件的目录
 
-  - [](software)
+  - [](ref:software)
```

Comment:
There is a file called `software.md` in the same directory, so we have to use a different name here.